pkg/cli/admin/addnodes/addnodes.go

```diff
@@ -0,0 +1,23 @@
+package addnodes
+
+import (
+	"github.com/spf13/cobra"
+
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+func NewCmdAddNodes(f kcmdutil.Factory, streams genericiooptions.IOStreams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add-nodes",
+		Short: "Commands for adding new nodes to an OpenShift cluster",
```

Comment:
Nit: `Command`

Reply:
This is the parent that groups a set of different (sub)commands - maybe we could use a description more similar to the `oc adm release` one (`This tool is used to...`)